metrics/metrics.go

```diff
@@ -0,0 +1,275 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/bnb-chain/greenfield-challenger/config"
+)
+
+const (
+	// Monitor
+	MetricGnfdSavedBlock = "gnfd_saved_block"
+	MetricGnfdSavedEvent = "gnfd_saved_event"
+	// increase(gnfd_saved_block[1m])
+	// increase(gnfd_saved_event[1m]) fix this
+
+	// Verifier
+	MetricVerifiedChallenges   = "verified_challenges"
+	MetricChallengeFailed      = "challenge_failed"
```

Comment:
Maybe we can also have a metrics to record the latest block `Height`? Then we can compare it with fullnode, to see if there is any lag.

Reply:
It is there.